pkg/chartutil/load.go

```diff
@@ -214,6 +215,68 @@ func LoadFile(name string) (*chart.Chart, error) {
 	return LoadArchive(raw)
 }
 
+// Getting Helm client home directory.
+
+func getHelmHomeDir() string {
+	var home string
+	if home = os.Getenv("$HELM_HOME"); home != "" {
+		return home + string(filepath.Separator)
+	}
+	if runtime.GOOS == "windows" {
+		home = os.Getenv("HOMEDRIVE") + os.Getenv("HOMEPATH")
+		if home == "" {
+			home = os.Getenv("USERPROFILE")
+		}
+	} else {
+		home = os.Getenv("HOME")
+	}
+	home += string(filepath.Separator) + ".helm" + string(filepath.Separator)
+
+	return home
+}
+
+// LoadIgnoreFiles loads global helmignore file from the default helm client
+// folder and if exists, helmignore file from the current folder into
+// a buffer and return a reader.
+func LoadIgnoreFiles(dir string) (*bytes.Reader, error) {
+	buff := bytes.NewBuffer(nil)
+
+	topdir, err := filepath.Abs(dir)
+	if err != nil {
+		return nil, err
+	}
+	topdir += string(filepath.Separator)
+
+	f, err := os.Open(topdir + ignore.HelmIgnore)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return nil, err
+		}
+	} else {
+		defer f.Close()
+		_, err = io.Copy(buff, f)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	homeDir := getHelmHomeDir()
```

Comment:
I got the idea of passing list of ignore files to LoadDir and it sounds great but I have a couple of questions:
1. Where do you see ignoreFiles string gets instantiated/populated? at the initialization time? Would be great to have a pointer which function would be the best candidate.
2. Since multiple files can be specified, I would like to use slice of strings.  
Appreciate your reply..